gpii/node_modules/matchMakerFramework/test/InverseCapabilitiesTests.js

```diff
@@ -19,7 +19,7 @@ https://github.com/GPII/universal/blob/master/LICENSE.txt
 "use strict";
 
 var fluid = require("infusion"),
-    jqUnit = fluid.require("jqUnit"),
+    jqUnit = fluid.require("node-jqunit", require, "jqUnit"),
```

Comment:
should probably just be  

jqUnit = fluid.registerNamespace("jqUnit"),

as you're doing a `kettle.loadTestingSupport();` further down